doomrainbow/src/main/java/com/detroitlabs/kyleofori/doomrainbow/RainbowView.java

```diff
@@ -119,15 +121,19 @@ private static void initDefaultCurrentLevelArcPaint() {
      */
     private float lambda = DEFAULT_CHILD_VIEW_ASPECT_RATIO;
 
-    public RainbowView(Context context) {
+    public RainbowView(final Context context) {
```

Comment:
It's a way to communicate intent - I don't plan to reassign this parameter within the scope of this constructor. If you mark things as `final` by default, then when you see a non-`final` parameter then you know that reassignment will be taking place in the following block of code.